pkgs/development/tools/omnisharp-roslyn/deps.nix

```diff
@@ -65,26 +65,26 @@
   }
```

Comment:
We should update this to the new style.

Reply:
This would require moving omnisharp-roslyn to `buildDotnetModules`. It's pretty tricky hovewer, since it uses non-nuget package sources, and requires their `url` to be preserved.

Luckily, I started work on this in a separate pull request :) - https://github.com/NixOS/nixpkgs/pull/178004